src/regedit/register.jl

```diff
@@ -504,8 +504,10 @@ function register(
         @debug("branch registry repo")
         git = gitcmd(registry_path, gitconfig)
         run(pipeline(`$git checkout -f master`; stdout=devnull))
-        run(pipeline(`$git branch -f $branch`; stdout=devnull))
-        run(pipeline(`$git checkout -f $branch`; stdout=devnull))
+        if branch != "master"
+            run(pipeline(`$git branch -f $branch`; stdout=devnull))
```

Comment:
Unrelated to this PR, but we could use `checkout -fb` instead of `branch` and `checkout` in separate commands.